opengrok-tools/src/test/python/test_commandSequence.py

```diff
@@ -77,6 +77,20 @@ def test_exit_2_handling():
     assert not cmds.failed
```

Comment:
please use snake case for the filename as that is best practice for python

Reply:
i.e. `test_command_sequence.py` ?

Reply:
yes